packages/utils/upgrade/src/cli/commands/run-codemods.ts

```diff
@@ -0,0 +1,76 @@
+import prompts from 'prompts';
+
+import inquirer from 'inquirer';
+import { loggerFactory } from '../../modules/logger';
+import { handleError } from '../errors';
+import * as tasks from '../../tasks';
+
+import type { Command } from '../types';
+import { Codemod } from '../../modules/codemod';
+
+export const runCodemods: Command = async (options) => {
+  try {
+    const { silent, debug, yes } = options;
+    const logger = loggerFactory({ silent, debug });
+
+    logger.warn(
+      "Please make sure you've created a backup of your codebase and files before upgrading"
+    );
+
+    const confirm = async (message: string) => {
+      if (yes) {
+        return true;
+      }
+
+      const { confirm } = await prompts({
+        name: 'confirm',
+        type: 'confirm',
+        message,
+      });
+
+      // If confirm is undefined (Ctrl + C), default to false
+      return confirm ?? false;
+    };
+
+    const selectCodemods = async (codemods: Codemod.VersionedCollection[]) => {
+      const selectableCodemods = codemods
+        .map(({ version, codemods }) =>
+          codemods.map((codemod) => ({
+            name: `(${version}) ${codemod.filename}`,
+            value: codemod,
+            checked: true,
+          }))
+        )
+        .flat();
+
+      const prompt = [
+        {
+          type: 'checkbox',
```

Comment:
Why not using an [autocompleteMultiselect](https://github.com/terkelg/prompts#autocompletemultiselectsame) instead?